File: 0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        for index, num in enumerate(nums):
            if (target - num) in hashmap:
                return [hashmap[target - num], index]
            hashmap[num] = index

File: 0001-two-sum/solution.go
func twoSum(nums []int, target int) []int {
    store := make(map[int]int)
    for i, num := range nums {
        if j, exists := store[target - num]; exists {
            return []int{i, j}
        }
        store[num] = i
    }
    return []int{}
}
